docker/docker-bake.hcl

```diff
@@ -43,6 +43,7 @@ target "core-build" {
   target = "build_env"
   contexts = {
     test_data = "./tests/data"
+    static_assets = "./assets"
```

Comment:
You need it in `target "core" {` core too (I can't comment there)

Reply:
Thanks, added in https://github.com/OpenRailAssociation/osrd/pull/8106/commits/e2ead0e450a800ab5df676b88272c85627a32f00 and fixed the CI 🙏